src/domain/hanke/list/Table.tsx

```diff
@@ -78,12 +79,18 @@ const Table: React.FC<TableProps> = ({ columns, data }) => {
           ))}
         </thead>
         <tbody {...getTableBodyProps()}>
-          {page.map((row) => {
+          {page.map((row, index) => {
             prepareRow(row);
             return (
               <tr {...row.getRowProps()}>
                 {row.cells.map((cell) => {
-                  return <td {...cell.getCellProps()}>{cell.render('Cell')}</td>;
+                  return (
+                    <td data-testid={`row${index}_cell_${cell.column.id}`} {...cell.getCellProps()}>
+                      {cell.column.id === 'Lopetus' || cell.column.id === 'Aloitus'
```

Comment:
Tämä toteutus ei sellaisenaan tue monikielisyyttä.
Ehkäpä tässä tapauksessa voisi ottaa nuo id:t käännöstiedostoista?
hankeList.tableHeader.endDate

Vai onko tuohon muuta mahkua käsitellä näitä kolumnien ID:itä kieliriippumattomasti?